### ISSUE TITLE
PHP 8.1 and higher incompatibility

### ISSUE THREAD

**Issue by nlasselmanr** [nilsteampassnet/TeamPass#3492]
### Steps to reproduce
1. Cloned github:master
2. Build container with dockerfile (cloned github today)
3. Ran the container
4. Completed the installation steps/wizard
5. Login with admin
6. settings->LDAP
7. Fill in the fields
8. Perform test config
9. In bottom right corner a dialog in progress shows but never closes and LDAP config not tested
10. Browser shows a 502 error on ldap.queries.php
11. container log shows PHP deprecated warning and error messages. 
12. Has to do with PHP 8.1 and higher.

### Expected behaviour
LDAP Test Config button should return a status

### Actual behaviour
Browser shows a 502 error on ldap.queries.php and container log shows PHP deprecated warning and error messages. 
Container log errors (there are many warnings and erros in the attached logs):
[error] 38#38: *233 FastCGI sent in stderr: "PHP message: PHP Deprecated:  Constant FILTER_SANITIZE_STRING is deprecated in /var/www/html/sources/checks.php on line 43
[error] 38#38: *240 FastCGI sent in stderr: "te = false): DateInterval, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/html/includes/libraries/Carbon/Traits/Difference.php on line 120
[error] 38#38: *240 upstream sent too big header while reading response header from upstream

### Server configuration
Container teampass running on Ubuntu 22.04 Microk8s 1.26
**Database: MariaDB 10.6**
**Teampass version: 3.0.0.22** 

### Client configuration
**Browser: Firefox 108.0.2 (64-bit)**
**Operating system: Windows 11**

### Logs
[teampass-768799f587-vdklc.log](https://github.com/nilsteampassnet/TeamPass/files/10429410/teampass-768799f587-vdklc.log)
[teampass.example.local_Archive [23-01-16 21-50-06].zip](https://github.com/nilsteampassnet/TeamPass/files/10429420/teampass.example.local_Archive.23-01-16.21-50-06.zip)

**Comment by hoanbc** [comment]
Agree, I use TeamPass v3.0.21 with PHP v8.2 and see same error when use LDAP Test Config.

**Comment by nilsteampassnet** [comment]
I currently develop Teampass with 8.1.
So I believe it is related to 8.2 as @hoanbc confirmed

**Comment by AKorolkovs** [comment]
Have same and much more issue running PHP 8.1. PHP7.4 works just fine

**Comment by nilsteampassnet** [comment]
Closed automatically due to inactivity (no response for over 1 year).
